core/sqf/sqenvcom.sh

```diff
@@ -29,8 +29,8 @@
 #  * Add an entry in sqf/LocalSettingsTemplate.sh to make path configurable.
 ##############################################################
 
-#Product version (Trafodion or derivative product)
-export TRAFODION_VER_PROD="Apache Trafodion "
+#Product version (Trafodion, Enterprise or Enterprise Adv)
```

Comment:
Merge conflict problem ! I had indeed corrected it in the last review. WIll revert to "derivative product".